animareflection/ui
# Controlled combobox data

## Description

##### Task link: N/A

- Made `Combobox` `groups` optional for situations where `content` override is used and preferred
- Added assertion if missing `content` or `groups` prop

## Test Steps

- Verify `Combobox` works well, with and without `groups` specified
- Verify assertion is thrown if `groups` or `content` is missing (I was going to add a test case for this, but our current test setup is monolithic for rendering 1 instance of a component, so we can do this later after refactoring the test suite to be more flexible)
